tests/datetime/test_comparison.py

```diff
@@ -237,21 +237,26 @@ def test_less_than_or_equal_with_timezone_false():
     assert not d1 <= d3
 
 
-def test_is_birthday():
+def test_is_anniversary():
```

Comment:
I would have parametrized this, or made 2 independent copies

Reply:
Two independent copies sound reasonable for me 👍